https-everywhere-checker

```diff
@@ -1 +1 @@
-Subproject commit 2ec94f1ce11a2046408a8df228447e2f0dd85cda
+Subproject commit ce35ca84ec75f3205b098d5de73ea293fc22f187
```

Comment:
Yeah, not sure exactly why, but doing the same thing on a (fresh) master seemed to do the trick, so I included this in #1169 instead. Thanks.